edm4eic.yaml

```
@@ -244,6 +244,22 @@ datatypes:
   ## ==========================================================================
   ## Calorimetry
   ## ==========================================================================
+  edm4eic::SimCalorimeterPulse:
+    Description: "Simulated calorimeter pulse prior to digitization, implements SimPulse."
```

```suggestion
    Description: "Simulated calorimeter pulse prior to digitization, implements SimPulse"
```

---

```suggestion
    Description: "Simulated calorimeter pulse prior to digitization"
```
Once we remove SimPulse